operatoringress/v1/types.go

```diff
@@ -70,9 +70,10 @@ type DNSRecordSpec struct {
 	// "Unmanaged".
 	//
 	// +kubebuilder:default:="Managed"
-	// +kubebuilder:validation:Optional
-	// +optional
-	DNSManagementPolicy DNSManagementPolicy `json:"dnsManagementPolicy"`
+	// +kubebuilder:validation:Required
+	// +required
```

Comment:
Delete this line please, it doesn't serve any purpose as it isn't picked up by any generators. `+kubebuilder:validation:Required` is the correct tag